docs/Security/Extended-Protection.md

```diff
@@ -98,8 +98,7 @@ Extended Protection **should not be enabled for hybrid servers that are publishe
 
 ### NTLMv1 is not supported when Extended Protection is enabled
 
-> [!NOTE]
-> To increase security, we recommend that you review and configure this setting regardless of whether you experience problems or not.
+**Note**: To increase security, we recommend that you review and configure this setting regardless of whether you experience problems or not.
```

Comment:
Let's do this instead

https://stackoverflow.com/questions/25654845/how-can-i-create-a-text-box-for-a-note-in-markdown

```
> **NOTE:** 
```

OR 

```
> **_NOTE:_** 
```

Reply:
Done.